src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -1410,7 +1410,8 @@ module Make_str (A : Wire_types.Concrete) = struct
               match spec.auth_type with
               | Proof ->
                   let vk =
-                    exists Side_loaded_verification_key.typ ~compute:(fun () ->
+                    exists (Side_loaded_verification_key.typ feature_flags)
```

Comment:
I believe (although the code doesn't make it clear) that these `feature_flags` correspond to the vk commitments for the *wrap* proof, so we should use `None` here.

Reply:
Done